### ISSUE TITLE
Interface generator fails for write-only properties

### ISSUE THREAD

**Issue by gotthardp** [python-sdbus/python-sdbus#42]
Hello,
I was generating code for a write-only property
```xml
<property name="Value" type="b" access="write"/>
```

Which causes this error:
```
  File "/usr/local/lib64/python3.11/site-packages/sdbus/interface_generator.py", line 437, in __init__
    raise ValueError(f"Unknown property access {self.is_read_only}")
                                                ^^^^^^^^^^^^^^^^^
AttributeError: 'DbusPropertyIntrospection' object has no attribute 'is_read_only'
```

This is because:
1) `access_type == 'write'` is not supported (although it is a valid value and used occassionally)
2) the `raise` tries to access `self.is_read_only` which is not set in the `else` branch

https://github.com/python-sdbus/python-sdbus/blob/1399e019634ce936434bc580c441b217b82f1826/src/sdbus/interface_generator.py#L439-L444

Could we have something like this instead, please?
```python
 if access_type == 'readwrite' or access_type == 'write': 
     self.is_read_only = False 
 elif access_type == 'read': 
     self.is_read_only = True 
 else: 
     raise ValueError(f"Unknown property access {access_type}")
```

**Comment by igo95862** [comment]
Hello,

Accessing `self.is_read_only` before it is assigned is definitely a bug. It is meant to be `access_type`.

By the way I am curious what service has a write only properties?

**Comment by gotthardp** [comment]
Thanks. Well, I designed a custom service with a write only property, although I admit now it makes no sense. But there is also https://github.com/abrt/reportd/blob/master/src/org.freedesktop.reportd.xml, just to prove I was not using it alone. :)

**Comment by igo95862** [comment]
Should be fixed in fa9e5efd6610e4fdf333715cfc097b7d40ce8e0e

BTW it looks like sd-bus does not support write only type properties. 

https://github.com/systemd/systemd/blob/8a76d3cc7029ea9add4d4ed38af72745f6ba7974/src/libsystemd/sd-bus/bus-introspect.c#L252

It can only emit the "readwrite" or "read" XML introspection.

**Comment by gotthardp** [comment]
Thank you!
